integrations/waffle-mars.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Waffle & Mars
+description: Learn how to use Waffle and Mars to write, compile, and deploy Ethereum smart contracts on Moonbeam.
+---
+
+# Using Waffle & Mars on Moonbeam
+
+![Waffle and Mars on Moonbeam](/images/waffle-mars/waffle-mars-banner.png)
+
+## Introduction
+
+[Waffle](https://waffle.io) and [Mars](https://ethereum-mars.readthedocs.io/en/latest/index.html) are two tools developers can use together to write, test, and deploy smart contracts onto Moonbeam. Waffle provides a lightweight JavaScript framework, that is TypeScript compatible, for writing and testing smart contracts. While Mars is a deployment manager that makes it easier to write and execute deployment configurations so you don't have to worry about your deployments.
+
+For the purposes of this guide, Waffle is used to compile your contracts to JSON that the EVM can understand. After compilation, Mars takes the JSON output and uses it to generate artifacts which are then used to deploy contracts to Moonbeam.
+
+In this guide, you'll be creating a TypeScript project to write and compile a smart contract using Waffle, then deploy it on to Moonbeam using Mars.
+
+## Checking Prerequisites
+
+--8<-- 'text/common/install-nodejs.md'
+
+As of writing of this guide, the versions used were 15.12.0 and 7.6.3, respectively.
+
+Waffle and Mars can be used on the Moonbase Alpha TestNet, but for the purposes of this guide, you will be deploying to a local Moonbeam development node. Check out the [Setting Up a Node](/getting-started/local-node/setting-up-a-node) documentation to learn how to spin up your own Moonbeam development node.
+
+## Create a TypeScript Project with Waffle & Mars
+
+To get started, you'll need to create a TypeScript project and install and configure a few dependencies such as TypeScript, Waffle, and Mars. You will also need to install Open Zeppelin Contracts as the contract you will be creating will use their ERC20 base implementation. The other library you'll need is TS Node, which will be used to execute the deployment script you'll create later in the guide. 
```

Comment:
You might as well deconstruct this long list of requirements into a bullet with some hyperlinks to where they can find or read more info.